tests/sentry/api/endpoints/test_project_events.py

```diff
@@ -1,95 +0,0 @@
-from __future__ import absolute_import
```

Comment:
This was pretty much a duplicate of https://github.com/getsentry/sentry/blob/master/tests/snuba/api/endpoints/test_project_events.py